### PR TITLE
rockchip: remove LED label hack

### PR DESCRIPTION
**Pull request by 1715173329**
Now we support parsing the color and function properties.

### CODE REVIEW

**Comment by 1715173329** [comment]
Updated description in patch 107 to avoid confusing.

**Comment by PolynomialDivision** [comment]
Can you reference to the commit that allows parsing the color and function properties in the commit message?

**Comment by 1715173329** [comment]
> Can you reference to the commit that allows parsing the color and function properties in the commit message?

Done ;)

File: target/linux/rockchip/patches-6.1/100-rockchip-use-system-LED-for-OpenWrt.patch [hunk 1]
```diff
@@ -27,25 +35,29 @@ Signed-off-by: David Bauer <mail@david-bauer.net>
  	};
  
  	chosen {
-@@ -49,18 +54,18 @@
+@@ -48,19 +54,22 @@
+ 		pinctrl-names = "default";
  
  		lan_led: led-0 {
++			color = <LED_COLOR_ID_GREEN>;
++			function = LED_FUNCTION_LAN;
```
Comment (PolynomialDivision) [inline target/linux/rockchip/patches-6.1/100-rockchip-use-system-LED-for-OpenWrt.patch:43]: Why doesn't the upstream dts file contain the color and function property?

Comment (1715173329) [reply target/linux/rockchip/patches-6.1/100-rockchip-use-system-LED-for-OpenWrt.patch:43]: When it was merged upstream it still used label binding.

File: target/linux/rockchip/armv8/base-files/etc/board.d/01_leds [hunk 2]
```diff
@@ -25,8 +25,8 @@ friendlyarm,nanopi-r5c)
 	;;
 friendlyarm,nanopi-r5s)
 	ucidef_set_led_netdev "wan" "WAN" "green:wan" "eth0"
```
Comment (PolynomialDivision) [inline target/linux/rockchip/armv8/base-files/etc/board.d/01_leds:27]: I'm sorry. I guess you are right, but I am thinking about why this works although we delete the `"green:wan"` label?

Comment (1715173329) [reply target/linux/rockchip/armv8/base-files/etc/board.d/01_leds:27]: > I am thinking about why this works although we delete the "green:wan" label?

We have `color` (green) and `function` (wan) properties ;)

**Pull request merged**